CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ This project employs a versioning scheme described in [RELEASE.md](RELEASE.md#ve
 
 ## [Unreleased]
 
+## [1.27.6]
+
+### Changed
```

Comment:
#701 is a bug fix in CI. It is better to use `Fixed`.

https://keepachangelog.com/en/1.1.0/#how

```suggestion
### Fixed
```

Reply:
Thank you. I corrected it.